sqlite3.go

```diff
@@ -19,6 +19,7 @@ package sqlite3
 #cgo CFLAGS: -DSQLITE_TRACE_SIZE_LIMIT=15
 #cgo CFLAGS: -DSQLITE_OMIT_DEPRECATED
 #cgo CFLAGS: -DSQLITE_DISABLE_INTRINSIC
+#cgo CFLAGS: -DSQLITE_ENABLE_PREUPDATE_HOOK
```

Comment:
Please move this to a file named `sqlite3_opt_preupdate_hook.go` and set a build tag `sqlite_preupdate_hook`

Because of possible performance penalities, this should be an option which a user should activate with a build tag.